crates/modelardb_server/src/remote.rs

```diff
@@ -669,6 +752,37 @@ impl FlightService for FlightServiceHandler {
                     body: batch_bytes.into(),
                 })
             }))))
+        } else if action.r#type == "UpdateRemoteObjectStore" {
+            // If the type of the new remote object store is not "s3" or "azureblobstorage", return an error.
+            let (object_store_type, offset_data) = extract_argument(&action.body)?;
+
+            let object_store = match object_store_type {
```

Comment:
Should it be possible to pass something like `None` to disable data transfer? This use case may be too niche to spend development time on it.

Reply:
We discussed making it possible to delete the object store to disable data transfer but it was determined that it was too niche to include in the main version of the system. Instead it should only be added if a customer specifically requests it.